ddtrace/_monkey.py

```diff
@@ -84,8 +84,29 @@
     "dogpile_cache": True,
     "yaaredis": True,
     "asyncpg": True,
+    "datadog_lambda": bool(os.environ.get("AWS_LAMBDA_FUNCTION_NAME", False)),  # patch only in AWS Lambda environments
 }
 
+
+def get_aws_lambda_module_to_patch():
```

Comment:
Can we always attempt to patch the `datadog_lambda` module and move this logic to ddtrace/contrib/datadog_lambda/patch.py:patch()?